src/V3Options.cpp

```diff
@@ -583,14 +583,7 @@ string V3Options::filePath(FileLine* fl, const string& modname, const string& la
 
 void V3Options::filePathLookedMsg(FileLine* fl, const string& modname) {
     static bool shown_notfound_msg = false;
-    if (modname.find("__Vhsh") != string::npos) {
-        std::cerr << V3Error::warnMoreStandalone()
-                  << "... Unsupported: Name is longer than 127 characters;"
-                  << " automatic file lookup not supported.\n";
-        std::cerr << V3Error::warnMoreStandalone()
-                  << "... Suggest putting filename with this module/package"
-                  << " onto command line instead.\n";
-    } else if (!shown_notfound_msg) {
```

Comment:
Ah yeah good point - I guess it should say `"automatic file lookup may not work due to file name length limits"` instead since sometimes it does work now!

Reply:
Done